src/cmd/cli/command/compose.go

```diff
@@ -489,99 +491,122 @@ func makeComposePsCmd() *cobra.Command {
 }
 
 func makeComposeLogsCmd() *cobra.Command {
-	logType := logs.LogTypeRun
 	var logsCmd = &cobra.Command{
 		Use:         "logs [SERVICE...]",
 		Annotations: authNeededAnnotation,
-		Aliases:     []string{"tail"},
 		Short:       "Show logs from one or more services",
-		RunE: func(cmd *cobra.Command, args []string) error {
-			var name, _ = cmd.Flags().GetString("name")
-			var etag, _ = cmd.Flags().GetString("etag")
-			var deployment, _ = cmd.Flags().GetString("deployment")
-			var raw, _ = cmd.Flags().GetBool("raw")
-			var since, _ = cmd.Flags().GetString("since")
-			var utc, _ = cmd.Flags().GetBool("utc")
-			var verbose, _ = cmd.Flags().GetBool("verbose")
-			var filter, _ = cmd.Flags().GetString("filter")
-			var until, _ = cmd.Flags().GetString("until")
+		RunE:        handleLogsCmd,
+	}
+	setupLogsFlags(logsCmd)
+	logsCmd.Flags().String("until", "", "show logs until duration/time")
+	logsCmd.Flags().Int32("limit", 0, "number of log lines to retrieve")                // NOTE: -n is already used by --name
```

Comment:
OK to remove `--name` and `-n`. We already support service names as positional args.